recipes/flac/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "flac" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/xiph/flac.git
+  git_rev: {{ version }}
```

Comment:
Thanks, I've fixed that by point to the url of the tar.gz of that release on github. However, the problems seems to pertain to sdl2_mixer, not flac:

~~~
Building  sdl2_mixer-2.0.1-vc10_0
Traceback (most recent call last):
  File "C:\Miniconda3\lib\site-packages\conda_build_all\builder.py", line 200, in build
    conda_build.api.build(meta.meta, config=config)
AttributeError: 'module' object has no attribute 'api'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  ....
PermissionError: [WinError 5] Access is denied: 'C:\\Miniconda3\\conda-bld\\work\\.git\\objects\\pack\\pack-1f3f954ec4500439b96cf1d242b9f3fcc1829707.idx'
~~~

Another thing I noticed, is that for Python 3.5, all packages are planned to be built twice:

~~~
Resolved dependencies, will be built in the following order: 
	mad-0.15.1b-vc14_0 (will be built: True)
	mad-0.15.1b-0 (will be built: True)
	ogg-1.3.2-vc14_0 (will be built: True)
	ogg-1.3.2-0 (will be built: True)
	smpeg2-2.0.0-vc14_0 (will be built: True)
	smpeg2-2.0.0-0 (will be built: True)
	vorbis-1.3.5-vc14_0 (will be built: True)
	vorbis-1.3.5-0 (will be built: True)
	flac-1.3.1-vc14_0 (will be built: True)
	flac-1.3.1-0 (will be built: True)
	sdl2_mixer-2.0.1-vc14_0 (will be built: True)
	sdl2_mixer-2.0.1-0 (will be built: True)
~~~

while this isn't happening for Python 3.4 or 2.7...